### PR TITLE
docs: update documentation for additionalRemoteWrite

### PR DESCRIPTION
**Pull request by rnishtala-sumo**
##### Description

Update documentation around additionalRemoteWrite for kube-prometheus-stack

### CODE REVIEW

File: deploy/docs/additional_prometheus_configuration.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ kube-prometheus-stack:  # For values.yaml
       # ...
       prometheusSpec:
         # ...
-        remoteWrite:
+        additionalRemoteWrite:
           # ...
```
Comment (sumo-drosiek) [inline deploy/docs/additional_prometheus_configuration.md:40]: no need for `# ...` it was indication that there was something to copy

also, we should search documentation for all `remoteWrite` mentions, as I'm not sure but we probably mention in multiple places that you need to copy it from our `values.yaml` which is no longer true

**Review by sumo-drosiek — COMMENTED** [review]
I would like to ask you also to go through the process of adding additional prometheus metrics, so we can have additional input on that, and make it less complicated (/more understable) :sweat:

File: deploy/docs/additional_prometheus_configuration.md (hunk 1 above)
Comment (rnishtala-sumo) [reply deploy/docs/additional_prometheus_configuration.md:40]: ok got it, will look for other references to remoteWrite. Also, I was able to send some metrics from the k8s-collector to a sumo instance. Will try adding additional prometheus metrics.

Comment (rnishtala-sumo) [reply deploy/docs/additional_prometheus_configuration.md]: get this error when I try to send Prometheus metrics to Sumo (long) via Fluentd.
config error file="/fluentd/etc/fluent.conf" error_class=Fluent::ConfigError error="Invalid SumoLogic endpoint url: "

it's not clear where in endpoint url gets configured in fluentd, configured the following in values.yaml
endpoint: "https://long-api......."

Comment (rnishtala-sumo) [reply deploy/docs/additional_prometheus_configuration.md]: Also @sumo-drosiek there doesn't seem to be any other mention of copying remoteWrite from values.yaml in the documentation

Comment (rnishtala-sumo) [reply deploy/docs/additional_prometheus_configuration.md]: Was able to send additional Prometheus metrics from a new metrics source. Also, removed copy references from the readme and added a note like below

NOTE: It is best practice to add custom configuration to a user-supplied values file and then use it like so (```helm install -f my_values.yaml -n sumologic```)

File: deploy/docs/additional_prometheus_configuration.md [hunk 2]
```diff
@@ -271,15 +268,13 @@ To send custom metrics to Sumo Logic you need to update it to include a rule to
 Make sure you include the same tag you created in your Fluentd configmap in the previous step.
 Here is an example addition to the configuration file that will forward metrics to Sumo:
 
+NOTE: Please add the below to a user-supplied values file  (```helm install -f```)
```
Comment (sumo-drosiek) [inline deploy/docs/additional_prometheus_configuration.md:271]: incomplete command

**Comment by sumo-drosiek** [comment]
Generally I'm not sure about copying the same note in multiple places in one document
Also for all our docs Note's are usually **bold**

cc: @astencel-sumo

File: deploy/docs/additional_prometheus_configuration.md (hunk 2 above)
Comment (rnishtala-sumo) [reply deploy/docs/additional_prometheus_configuration.md:271]: ok makes sense, I could keep the one comment at the beginning and remove the rest of them. Would that work?

**Review by sumo-drosiek — APPROVED** [review]
Thanks!

**Comment by swiatekm-sumo** [comment]
@rnishtala-sumo the CI should be fine if you rebase on `main`.

**Comment by rnishtala-sumo** [comment]
Fixes issue: #1822

**Pull request merged**